tenets/codelingo/code-review-comments/use-crypto-rand/codelingo.yaml

```diff
@@ -0,0 +1,39 @@
+funcs:
+  - name: isKeyName
+    type: asserter
+    body: |
+      function(name) {
+        return name.toLowerCase().indexOf("key") !== -1
+      }
+tenets:
+  - name: use-crypto-rand
+    flows:
+      codelingo/review:
+        comment: |
+          Do not use package math/rand to generate keys, even 
+          throwaway ones. Unseeded, the generator is completely predictable. 
+          Seeded with time.Nanoseconds(), there are just a few bits of entropy. 
+          Instead, use crypto/rand's Reader, and if you need text, print to 
+          hexadecimal or base64
+    query: |
+      import codelingo/ast/go
+
+      go.import_spec(depth = any):     
+        go.basic_lit:
+          value == "math/rand"
+      any_of:
+        @review comment
```

Comment:
I remember you were having trouble with multiple review comments inside of an any_of. This seems to be not complaining about having multiple review comments anymore, which is good. However, it looks like you're getting multiple hits for the same thing. I assume this is because you have two review comments. Is it possible to move the review comment to a 'pipeline' such as any_of.. A question for  Blake

```
Running review flow...
example-bad.go:19:2

    Do not use package math/rand to generate keys, even
    throwaway ones. Unseeded, the generator is completely predictable.
    Seeded with time.Nanoseconds(), there are just a few bits of entropy.
    Instead, use crypto/rand's Reader, and if you need text, print to
    hexadecimal or base64


    ...
    )

  + func Key() string {
  +     buf := make([]byte, 16)
  +     _, err := rand.Read(buf)
  +     if err != nil {
  +             panic(err) // out of randomness, should never happen
  +     }
  +     return fmt.Sprintf("%x", buf)
  +     // or hex.EncodeToString(buf)
  +     // or base64.StdEncoding.EncodeToString(buf)
  + }

    ...

[o]pen [k]eep [d]iscard : k
example.go:19:2

    Do not use package math/rand to generate keys, even
    throwaway ones. Unseeded, the generator is completely predictable.
    Seeded with time.Nanoseconds(), there are just a few bits of entropy.
    Instead, use crypto/rand's Reader, and if you need text, print to
    hexadecimal or base64


    ...
    )

  + func Key() string {
  +     buf := make([]byte, 16)
  +     _, err := rand.Read(buf)
  +     if err != nil {
  +             panic(err) // out of randomness, should never happen
  +     }
  +     return fmt.Sprintf("%x", buf)
  +     // or hex.EncodeToString(buf)
  +     // or base64.StdEncoding.EncodeToString(buf)
  + }

    func main() {
    ...

[o]pen [k]eep [d]iscard : k
```